ufs2arco/regrid/ufsregridder.py

```diff
@@ -182,8 +182,12 @@ def regrid_tripolar(
                 interp_v = rg_vt(ds_in[var2])
 
                 # rotate to earth-relative
-                urot = interp_u * ds_rot.cos_rot + interp_v * ds_rot.sin_rot
-                vrot = interp_v * ds_rot.cos_rot - interp_u * ds_rot.sin_rot
+                urot = (
+                    interp_u * ds_rot.cos_rot.values + interp_v * ds_rot.sin_rot.values
+                )
+                vrot = (
+                    interp_v * ds_rot.cos_rot.values - interp_u * ds_rot.sin_rot.values
+                )
```

Comment:
Out of curiosity, is there a reason for adding the `.values` accessor here?

Reply:
I think it just made it go faster... for some reason it was taking a sec on my end.

Reply:
I actually went and refactored cos/sin a little in general to make it neater so this went back to the old way (without .values)

Reply:
Ah that makes sense. I guess they were dask arrays originally. Thanks for looking into that